packages/utils/src/resolve-additional-resolvers.ts

```diff
@@ -321,148 +319,18 @@ export function resolveAdditionalResolvers(
 
         return resolvers;
       } else {
-        const baseOptions: any = {};
```

Comment:
Why do we remove this?

Reply:
The code seems to be duplicated (I have checked with a diff tool to be sure 😛) so I have replaced the duplication by a function call. I can revert if you think it is relevant to have a duplication here.

Reply:
Oh ok ok :D